src/plugins/PluginsWrapper.ts

```diff
@@ -2,18 +2,48 @@ import {InitializationOptions, MonitoredPlugin, OnFailureOptions, OnStartOptions
 
 export class PluginsWrapper implements Omit<MonitoredPlugin, 'initialize'> {
     constructor(private readonly plugins: MonitoredPlugin[], opts: InitializationOptions) {
-        plugins.forEach((p) => p.initialize?.(opts));
+        plugins.forEach(p => p.initialize?.(opts));
     }
 
     onStart(opts: OnStartOptions): void {
-        this.plugins.forEach((p) => p.onStart(opts));
+        this.plugins.forEach(p => p.onStart(opts));
     }
 
     onSuccess(opts: OnSuccessOptions): void {
-        this.plugins.forEach((p) => p.onSuccess(opts));
+        this.plugins.forEach(p => p.onSuccess(opts));
     }
 
     onFailure(opts: OnFailureOptions): void {
-        this.plugins.forEach((p) => p.onFailure(opts));
+        this.plugins.forEach(p => p.onFailure(opts));
+    }
+
+    async increment(name: string, value: number, tags?: {[key: string]: string} | string[]): Promise<void> {
+        for (let index = 0; index < this.plugins.length; index++) {
```

Comment:
More simpler:
```
     for (const p of this.plugins) {
            await p.increment(name, value, tags);
        }
```

Reply:
As below, I suggest turning this into `Promise.all(plugins.map(...))`, no need to be sequential here

Reply:
whatever way you choose make those methods the same... this code doesn't have a standard at the moment and I see 3 different ways